corehq/ex-submodules/casexml/apps/phone/tests/test_sync_log_assertions.py

```diff
@@ -29,3 +29,24 @@ def test_update_dependent_case(self):
             # this test just ensures it doesn't still do that.
             for log in [sync_log, SimplifiedSyncLog.from_other_format(sync_log)]:
                 log.update_phone_lists(xform, [parent_case])
+
+    def test_update_dependent_case_owner_still_present(self):
```

Comment:
looks like this test only runs on the old code, so curious why you bothered with the `test_only_get_dependent_cases_on_phone` thing. also - have you verified this works ok on new code?